# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wdiff/hotfix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Hmm, I think this is simply screwed ifdef logic (first of all,
getdelim is never tested, so HAVE_GETDELIM is never defined, second
the gnu clib has both, getdelim and getline, defined already).
	--jsaw

--- wdiff-0.5g/lib/getline.c.orig	2006-06-16 15:34:31.000000000 +0200
+++ wdiff-0.5g/lib/getline.c	2006-06-16 15:36:28.000000000 +0200
@@ -31,7 +31,8 @@
 #include <stdio.h>
 #include <sys/types.h>
 
-#if defined __GNU_LIBRARY__ && HAVE_GETDELIM
+#if defined __GNU_LIBRARY__
+#elif defined HAVE_GETDELIM
 
 int
 getline (lineptr, n, stream)
